spinnman/transceiver.py

```diff
@@ -1253,6 +1253,68 @@ def get_iobuf(self, core_subsets=None):
         process = ReadIOBufProcess(self._scamp_connection_selector)
         return process.read_iobuf(self._iobuf_size, core_subsets)
 
+    def set_watch_dog_mode_at_chip(self, watch_dog_mode, chip_x, chip_y):
+        """ changes the chips watchdog parameter to watch_dog_mode
+
+        :param watch_dog_mode: the mode / value to change the watchdog value\
+         in sv to
+         :type watch_dog_mode: int or bool
+        :param chip_x: chip x coord to write new watchdog param to
+        :type chip_x: int
+        :param chip_y: chip y coord to write new watchdog param to
+        :type chip_y: int
+        :rtype: None 
+        """
+
+        # get actual watchdog value from machine
+        actual_watch_dog_value = self._get_sv_data(
+            chip_x, chip_y, SystemVariableDefinition.software_watchdog_count)
```

Comment:
It probably isn't worth checking this - just set it to what the user suggests...